imitation/util/rollout.py

```diff
@@ -312,25 +356,28 @@ def generate_multiple(policies, env, n_timesteps, *, truncate=True):
           episode completion states, the last obs-state-obs triple in every
           episode is omitted. (See GitHub issue #1)
       env (gym.Env): The environment the policy should act in.
-      n_timesteps (int): The number of obs-action-obs
-          triples to generate. If the number of policies given doesn't
-          divide this number evenly, then the last policy generates
-          more timesteps.
-      truncate (bool): should collected episodes be truncated so that *only*
-          `n_timesteps` are returned? Otherwise this will potentially return
-          more timesteps.
+      n_timesteps (int): The minimum number of obs-action-obs-reward tuples to
+          collect (may collect more if episodes run too long). Set exactly one
+          of `n_timesteps` and `n_episodes`, or this function will error.
+          If the number of policies given doesn't divide this number evenly,
+          then the last policy generates more timesteps than the other policies.
+      truncate (bool): If True and n_timesteps is not None, then drop any
+          additional samples to ensure that exactly `n_timesteps` samples are
+          returned.
 
   Returns:
       rollout_obs_old (array): A numpy array with shape
-          `[n_timesteps] + env.observation_space.shape`. The ith observation in
+          `[n_samples] + env.observation_space.shape`. The ith observation in
```

Comment:
Are you sure you want this to be `n_samples` and not `n_timesteps` for consistency with the `n_timesteps` arg?

Reply:
It isn't always `n_timesteps` because we can go over. I'll add this as a clarification

Reply:
(note: also do this for `generate()`)